charts/sourcegraph/values.yaml

```diff
@@ -221,6 +221,15 @@ frontend:
     create: true
     name: sourcegraph-frontend
 
+migrator:
+  enabled: true
+  image:
+    defaultTag: insiders@sha256:18fb43c040e91bce06d8123e0f859cdfb1eded897f5c0b3a138864d9a885c1eb
+    name: "migrator"
+  env: {}
+  resources: {}
+  podSecurityContext: {}
```

Comment:
This should default to the same context as in this file: https://github.com/sourcegraph/deploy-sourcegraph/blob/master/overlays/non-privileged/frontend/sourcegraph-frontend.Deployment.yaml#L10